end2end/metric.py

```diff
@@ -15,6 +15,25 @@ def add(self, value):
             self.value = value * self.alfa + (1 - self.alfa) * self.value
 
 
+class Sample(object):
+    def __init__(self):
+        self.sample_size = os.getenv('PERCENTILE_SAMPLE_SIZE', 1000)
+        self.sample = []
+        self.p95 = None
+        self.p98 = None
+        self.p99 = None
+
+    def add(self, value):
+        size = len(self.sample)
+        if size == self.sample_size:
+            sorted_sample = sorted(self.sample);
+            self.p95 = sorted_sample[int(math.ceil((size * 95.99) / 100)) - 1]
```

Comment:
@rcillo I believe python arrays start from index 0. For example we take array of 10 elements and want to calculate 90% percentile then according to formula we get:
`(10 * 90 / 100) - 1 = 8` which is 9th element of the array. does it look right?